plugins/in_tail/win32/stat.c

```diff
@@ -35,6 +35,8 @@
  */
 
 #define UINT64(high, low) ((uint64_t) (high) << 32 | (low))
+#define LDAP_TO_SECONDS_DIVISOR 10000000
+#define LDAP_TO_EPOCH_DIFF_SECONDS 11644473600
```

Comment:
Please rename this constant so it's clear that it reflects the delta between windows' value for epoch and unixs' value for epoch.